Reenable autot_axis tests. Keep max Y to 500.

danlipsa opened the pull request.

danlipsa commented:
Data in
https://github.com/UV-CDAT/uvcdat-testdata/pull/131

danlipsa commented:
@doutriaux1 @aashish24 Please review

doutriaux1 commented:
@danlipsa is it ok now on mac? The original comment said it crashes on mac, why is it not crashing anymore?

doutriaux1 commented on testing/vcs/test_vcs_autot_axis_titles.py:
```diff
@@ -18,8 +18,8 @@
+    ySize = 500
 else:
     xSize = 800
     ySize = 400
```
don't you need `250` and `500` here as well?

danlipsa replied:
The problem way the Y size was bigger than the screen size. The way it is now works fine on the lowest resolution on my mac: 1024 x 640

aashish24 replied:
@danlipsa what is the logic of picking these hard-coded x and y size?

danlipsa replied:
@aashish24 I am making the window tall and narrow or long and thin so that ratio=autot is will be very different than the old behavior where the data is scaled with the window.

danlipsa commented:
@doutriaux1 The tests worked on my mac because I had higher resolution. I switched to the lowest resolution and some of the tests failed - I fixed that by reducing the size of the images generated. I think the problem is related to
https://github.com/UV-CDAT/uvcdat/issues/1740

danlipsa commented:
@doutriaux1 @aashish24 ping

aashish24 commented:
LGTM 👍
